Project/src/main/resources/templates/default.html

```diff
@@ -2,23 +2,11 @@
 <html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <title>Home Page</title>
 </head>
 <body>
     <header th:insert="header.html :: header"></header>
-    <h1 th:inline="text">Hello [[${#httpServletRequest.remoteUser}]]!</h1>
-    <h2>User Info</h2>
-    <table>
-        <tr th:each="user : ${user}">
-            <td th:text="${user.username}"></td>
-            <td th:text="${user.password}"></td>
-            <td th:text="${user.role.toString()}"></td>
-        </tr>
-    </table>
-
-    <form th:action="@{/logout}" method="post">
-        <input type="submit" value="Sign Out"/>
-    </form>
+    <h1>Hello! Your account seems to be bugged - Please register again!</h1>
```

Comment:
Maybe we should consider throwing an error/going to an error page instead? This view isn't really all that useful anymore.

Reply:
Youre right